src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -110,134 +113,68 @@ public ArmSubsystem() {
         }
 
         // Set default PID values elbow
-        for (var config : new PID[]{ELBOW_DEFAULT, ELBOW_STEADY}) {
+        for (var config : new PID[] {ELBOW_DEFAULT, ELBOW_STEADY}) {
             m_elbow.getPIDController().setP(config.P, config.slot);
             m_elbow.getPIDController().setI(config.I, config.slot);
             m_elbow.getPIDController().setD(config.D, config.slot);
             m_elbow.getPIDController().setFF(config.FF, config.slot);
             m_elbow.getPIDController().setDFilter(config.DF, config.slot);
         }
 
+        m_elbow.enableSoftLimit(SoftLimitDirection.kForward, true);
+        m_elbow.enableSoftLimit(SoftLimitDirection.kReverse, true);
+        m_shoulder.enableSoftLimit(SoftLimitDirection.kForward, true);
+        m_shoulder.enableSoftLimit(SoftLimitDirection.kReverse, true);
+
         // Set the soft limits
         m_shoulder.setSoftLimit(CANSparkMax.SoftLimitDirection.kForward, 60);
         m_shoulder.setSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, -15);
         m_elbow.setSoftLimit(CANSparkMax.SoftLimitDirection.kForward, 60);
         m_elbow.setSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, -15);
+    }
 
+    public void setState(State desiredState) {

```

Comment:
This needs to be dropped.

Reply:
Sorry, how else would you set the state?

Reply:
*reverted. This should be reverted.